src/OrchardCore.Modules/OrchardCore.Users/Controllers/AdminController.cs

```diff
@@ -1,4 +1,5 @@
 using System.Security.Claims;
+using GraphQLParser;
```

Comment:
Why?

Reply:
I'm not sure, seems it has been added accidently :)